tasks/bump.js

```diff
@@ -56,6 +56,16 @@ module.exports = function(grunt) {
         exactVersionToSet = false;
     }
 
+
+    //Use: --commitmsg="add me to commit message"
+    var appendToCommitMessage = grunt.option('commitmsg');
```

Comment:
Except that there's already an option for that and I'm adding functionality, not replacing it. I agreed with the v1.2.3 approach and didn't see any reason to override it or force someone to choose between one commit message approach over the other. Personally, I just wanted to add detail to my commit message.